docs/FAQ.md

```diff
@@ -13,5 +13,11 @@ When you run `arduino-cli board list`, your board doesn't show up. Possible caus
 For a deeper understanding of how FQBN works, you should understand
 the [Arduino platform specification][0].
 
+##  How to set multiple board options?
+
+Additional board options have to be separated by commas (instead of colon):
+
+`$ arduino-cli compile -b "LGT8fx Boards:avr:328:clock=16MHz,variant=modelP" test-sketch`
```

Comment:
```suggestion
`$ arduino-cli compile --fqbn "esp8266:esp8266:generic:xtal=160,baud=57600" TestSketch`
```
Use of the long option `--fqbn` makes the command more self-documenting than the short option. New users of Arduino CLI may not have heard the term "FQBN" before, so this helps to reinforce the concept.

---
I believe the ESP8266 core for Arduino is the most popular platform which has multiple board options. So this example will be more universally recognizable. The LGT8fx Boards platform is less popular and having a space in the vendor name is very rare.

---
Use of the established CamelCase sketch naming convention (`TestSketch` vs `test-sketch`) will make that section of the command more recognizable.